docs/reference/cat/nodeattrs.asciidoc

```diff
@@ -1,8 +1,77 @@
 [[cat-nodeattrs]]
 === cat nodeattrs
 
-The `nodeattrs` command shows custom node attributes.
-For example:
+Returns information about custom node attributes.
+
+[[cat-nodeattrs-api-request]]
+==== {api-request-title}
+
+`GET /_cat/nodeattrs`
+
+[[cat-nodeattrs-api-query-params]]
+==== {api-query-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=http-format]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=cat-h]
++
+See <<cat-nodeattrs-api-columns>> for a list of available columns. See
```

Comment:
If I were writing this, I might just embed the column types in this section (i.e. as definition list that uses ::: ).

Reply:
Relocated these with 925d6b5.